### ISSUE TITLE
SC 2.4.11 Character Key Shortcuts - Proposal for change of wording

### ISSUE THREAD

**Issue by MakotoUeki** [w3c/wcag21#669]
-- Current wording --

Success Criterion 2.4.11 Character Key Shortcuts

If a keyboard shortcut consisting entirely of one or more character keys is implemented in content, then a mechanism is available to turn it off or to remap it to a shortcut that can use at least one non-character key, unless the keyboard shortcut for a user interface component is only active when that component has focus.

-- Suggested revisions --

Success Criterion 2.4.11 Character Key Shortcuts

A keyboard shortcut consisting entirely of one or more character keys can be turned off or remapped to another shortcut that contains one or more non-character keys, unless the shortcut is active only when the associated component has focus.

--

The intention of the revisions is to simplify the sentence structure. The sentence structure “If A is B, then C is D” tends to become long and redundant, making it confusing and hard to follow and translate.

**Comment by awkawk** [comment]
(official WG response)
Thank you for the comment. In response to other issues (386, 659, 660) we have changed the structure of this SC. We believe that the new structure is clearer and hope it addresses your need for easier translation.
